### PR TITLE
Add support for replaying network requests from a captured HAR file

### PR DESCRIPTION
**Pull request by nwmac**
Adds helper scripts for replaying network requests from, HAR files.

For developer debugging use.

See included developer documentation.

### CODE REVIEW

File: docs/developer/har-files.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+Where:
+
+- `<har_file>` is the path to the HAR file to use
+- `<version>` is the version number of the UI to use (eg. `v2.8.0`, `v2.7.6`, `latest`)
+
+You'll get a similar print out to above, but the UI is loaded from the production build.
+
+## Exporting requests/responses
+
+You may want to inspect the network requests/responses contained in a HAR file. To support this, you can export exch request into a separate json file via the `har-export` script, eg.
```
Comment (aalves08) [inline docs/developer/har-files.md:59]: nitpick: `exch` should be `each`

**Review by aalves08 — COMMENTED** [review]
Just a minor nitpick on otherwise amazing work... 👏 👏 👏  This is going to be really useful for us developers. Thank you 🙇

File: docs/developer/har-files.md (hunk 1 above)
Comment (nwmac) [reply docs/developer/har-files.md]: Fixed

**Review by aalves08 — APPROVED** [review]
LGTM

**Pull request merged**